commands/src/main/java/org/wildfly/extras/creaper/commands/security/AddAuthorizationModule.java

```diff
@@ -0,0 +1,168 @@
+            TimeoutException, InterruptedException {
+        Operations ops = new Operations(occ.client);
+        Address authorizationClassicAddress = Address.subsystem("security")
+                .and("security-domain", securityDomainName)
+                .and("authorization", "classic");
+        try {
+            boolean exists = ops.exists(authorizationClassicAddress);
+            if (!exists) {
+                ops.add(authorizationClassicAddress);
+            }
+        } catch (OperationException e) {
+            throw new IOException("Failed to access or create authorization=classic in security domain "
+                    + securityDomainName, e);
+        }
+
+        Address authorizationModuleAddress = authorizationClassicAddress.and("policy-module", name);
+
+        if (replaceExisting) {
+            try {
+                ops.removeIfExists(authorizationModuleAddress);
+            } catch (OperationException e) {
+                throw new IOException("Failed to remove existing authorization module " + name + " in security domain "
+                        + securityDomainName, e);
+            }
+        }
+
+        ops.add(authorizationModuleAddress, Values.empty()
+                .andOptional("code", code)
+                .andOptional("flag", flag)
+                .andOptional("module", module)
+                .andObjectOptional("module-options", Values.fromMap(moduleOptions))
+        );
+    }
+
+    @Override
+    public void apply(OfflineCommandContext occ) throws CommandFailedException, IOException {
+        occ.client.apply(GroovyXmlTransform.of(AddAuthorizationModule.class)
+                .subtree("securitySubsystem", Subtree.subsystem("security"))
+                .parameter("atrSecurityDomainName", securityDomainName)
+                .parameter("atrName", name)
+                .parameter("atrCode", code)
+                .parameter("atrFlag", flag)
+                .parameter("atrModule", module)
+                .parameter("atrModuleOptions", moduleOptions)
+                .parameter("atrReplaceExisting", replaceExisting)
+                .build());
+    }
+
+    public static final class Builder {
+
+        private String securityDomainName;
+        private String name;
+        private String code;
+        private String flag;
+        private String module;
+        private final Map<String, String> moduleOptions = new LinkedHashMap<String, String>();
+        private boolean replaceExisting;
+
+        /**
+         * In case when you use this constructor then authorization module name is the same as its code.
+         */
+        public Builder(String code) {
+            this(code, code);
+        }
+
+        public Builder(String code, String name) {
+            if (code == null) {
+                throw new IllegalArgumentException("Code of the authorization module must be specified as non null "
+                        + "value");
```

Comment:
No need to do this, long strings are fairly common and the checkstyle configuration accounts for them, so they shouldn't trigger a warning.